iterating-over-data/iterating-over-dictionaries.md

```diff
@@ -34,9 +34,9 @@ for my_key, my_value in my_dict.items():
 
 ### About `.items()`
 
-The most unique part about iterating over dictionaries is that we should call `.items()` next to each dictionary when starting the for loop.
+The most unique part about iterating over the keys & values of dictionaries, is that we should call `.items()` next to each dictionary when starting the for loop.
```

Comment:
It feels a little odd to me to say "call `.items()` **next to** each dictionary". Would saying "call the method `.items()` **on** each dictionary" convey the same meaning?

Reply:
What about this? @yangashley @kelsey-steven-ada 

```suggestion
The most unique part about iterating over the keys & values of dictionaries is that we can call `.items()` on a dictionary to access the key/value pairs in a loop.
```

Reply:
Agreed, that reads better to me. I'll update it =]